Transformations: Support timezone when converting to time (#128553)
grafana/grafana: docs/sources/visualizations/panels-visualizations/query-transform-data/transform-data/index.md

    - It will parse the numeric input as a Unix epoch timestamp in milliseconds.
      You must multiply your input by 1000 if it's in seconds.
    - Will show an option to specify a DateFormat as input by a string like yyyy-mm-dd or DD MM YYYY hh:mm:ss

>     - The **Timezone** option determines how Grafana interprets input strings without timezone information. If unset, Grafana uses your configured default timezone, which defaults to the browser timezone.

Reviewer: ```suggestion
    - The **Timezone** option determines how Grafana interprets input strings without timezone information. If not set, Grafana uses the browser timezone or your configured default timezone.
```

Reviewer: @tamen25 - I've made the edits here, but this content needs to be actually added to `public/app/features/transformers/docs/content.ts` and then pushed to this file. See the directions at the beginning of this file.

cc @baldm0mma
